src/ildasm/dis.cpp

```diff
@@ -315,7 +315,6 @@ BOOL enumEHInfo(const COR_ILMETHOD_SECT_EH* eh, IMDInternalImport *pImport, DWOR
     COR_ILMETHOD_SECT_EH_CLAUSE_FAT ehBuff;
     const COR_ILMETHOD_SECT_EH_CLAUSE_FAT* ehInfo;
     BOOL    fTryInCode = FALSE;
-    if(g_ehInfo) VDELETE(g_ehInfo);
```

Comment:
I had it set to `0xfd' when it crashed. But I now realize that I thought it was freeing the `ehInfo' local and not `g_ehInfo'. I'll have a closer look at where this is incorrectly set.